internal/uvm/create_lcow.go

```diff
@@ -137,7 +137,7 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		KernelBootOptions:       "",
 		EnableGraphicsConsole:   false,
 		ConsolePipe:             "",
-		SCSIControllerCount:     1,
+		SCSIControllerCount:     4,
```

Comment:
Can we move `SCSIControllerCount` from `OptionsLCOW` to `Options`, so it can be used for both Windows and Linux UVMs?